packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Carbon from '../../../../carbon-graphs/src/js/carbon';
+import '../../css/Graph.module.scss';
+import '../../css/LineGraph.module.scss';
+import utils from '../../../../carbon-graphs/src/js/helpers/utils';
+
+class LineGraph extends React.Component {
```

Comment:
we can type check props using `prop-types`. here's an example in one of our components: https://github.com/cerner/terra-application/blob/main/src/application-base/ApplicationBase.jsx#L33

https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Added propTypes [here](https://github.com/cerner/terra-graphs/pull/74/commits/631c1818a81503e72ba2f63e2f41eb9df667ba04#diff-5e17603406b642035820d408335769ae48391021a7787fd86404b130590e9606R8-R12).